docs/UserGuide.md

```diff
@@ -3,56 +3,84 @@ layout: page
-commands to manage your client information and view your notes much faster 
+* Table of Contents
+{:toc}
+
+--------------------------------------------------------------------------------------------------------------------
+## 1. Introduction
+
+DonnaFin.io is a desktop application used by financial advisors to keep track of their
+client information and related tasks. This is a Java program with about 10 kloc with
+a JavaFX GUI. The '.io' in our name is a reflection of our belief that you deserve a
+faster workflow for input and output. If you can type fast, you can use our CLI-like
+commands to manage your client information and view your notes much faster
 than your typical customer relationship manager apps.
 
 Note: From here on we shall refer to DonnaFin.io as DonnaFin for your readability.
 
-* Table of Contents
-{:toc}
+### 1.1 Is this guide for you?
 
---------------------------------------------------------------------------------------------------------------------
+### 1.2 How to use the user guide
 
-## Quick start
+## 2. Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
 1. Download the latest `donnafin.jar` from [here](https://github.com/AY2122S1-CS2103T-W16-1/tp/releases).
 
 1. Copy the file to the folder you want to use as the _home folder_ for your DonnaFin app.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note that DonnaFin comes with some sample clients.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the DonnaFin.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a client named `John Doe` to the DonnaFin.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`view`**`3` : Opens the client window, where you can access the details of the 3rd client shown in the current list.
+   
+   * **`tab`**`a` : Access the assets tab of client that is currently viewed.
+   
+   * **`append`**`n/Good Class Bungalow ty/Property v/$10000000 r/newly bought with bank loan` : 
+   As we are in the Asset tab now, this will add an asset to the currently viewed client.
+   * **`remove`**`1` : Removes the 1st asset in the Asset tab.
+   
+   * **`home`** : Return to home window.
+   
+   * **`delete`**`3` : Deletes the 3rd client shown in the current list.
 
-   * **`clear`** : Deletes all contacts.
+   * **`clear`** : Deletes all clients.
```

Comment:
Would it be confusing to list non-global commands here in the 'examples of commands to use'? it may be a UG bug cuz they won't be able to do this in when they open the app and the commands listed don't make it clear that some of these examples work only in certain places. We maybe could just show list, home and view and leave everything else to the actual section on comands?

Reply:
Resolved to move through UG faster. @sheehui  pls fix this in a separate PR.